src/sources/prometheus/remote_write.rs

```diff
@@ -115,10 +115,9 @@ impl HttpSource for RemoteWriteSource {
         _header_map: HeaderMap,
         _query_parameters: HashMap<String, String>,
     ) -> Result<Vec<Event>, ErrorMessage> {
-        let byte_size = body.len();
```

Comment:
In this case, `body` is `Bytes`, which has the length known upfront. No reason not to send pass this data to the metric.